### DISCUSSION TITLE
Some series are divided in 2 parts in TVDB and also are available in 2 parts

### DISCUSSION THREAD

**Discussion by CrocodileDuty** [Taxel/PlexTraktSync#1502] [Q&A]
I got some episodes of "The office (Season 4 episode 15 to 19, Season 6 episode 27 & 26)", "Friends (Season 2 episode 13, Season 6 episode 16, Season 8 episode 24)", & 
"Brooklyn nine nine (Season 8 episode 10)". Example pics from TVDB, Trakt and My Plex Server.
![image](https://github.com/Taxel/PlexTraktSync/assets/32086892/f1492edc-3b3b-455c-a6ab-ed1b6357825b)
![image](https://github.com/Taxel/PlexTraktSync/assets/32086892/44b3be9a-7ce7-40f6-9181-c25c384dd420)
![image](https://github.com/Taxel/PlexTraktSync/assets/32086892/f65eed27-d019-4f19-bcd1-14aec8318f66)

What should I do to avoid this warnings like this for episodes?
![image](https://github.com/Taxel/PlexTraktSync/assets/32086892/62b8ad45-0b0d-45ec-97be-ff01441146f9)

**Comment by simonc56** [comment]
Nothing you can do about it.
Websites don't index the same way when there is a double episode.
Recommandation is to follow TMDB/Trakt numbering as much as possible.

Example for Brooklyn Nine-Nine :
https://www.themoviedb.org/tv/48891-brooklyn-nine-nine/discuss/6144b753ed28b9008e9f874d

**Comment by simonc56** [comment]
> What should I do to avoid this warnings like this for episodes?

Use logging filter in config.yml

**Reply by CrocodileDuty** [reply]
![Screenshot_20230621-083103_Termius.png](https://github.com/Taxel/PlexTraktSync/assets/32086892/2347bc2c-5791-45fa-8f65-7493b5ba6a39)

How & what should I edit here?

**Reply by CrocodileDuty** [reply]
Also can I use this to suppress "not found on trakt" warning on only specific shows and not to the shows which I havent tried to fix?

**Reply by simonc56** [reply]
To remove warnings about Friends show :
```
- level: WARNING
  message: ":Friends-"
```

**Reply by CrocodileDuty** [reply]
> To remove warnings about Friends show :
> 
> ```
> 
> - level: WARNING
> 
>   message: ":Friends-"
> 
> ```

And for multiple shows can we use like this?

```

  - level: WARNING

    message: ":Friends-"
    message: ":Brooklyn-"
    message: ":The-Office-"
 
```

**Reply by simonc56** [reply]
Test it and tell us.

**Reply by CrocodileDuty** [reply]
Doesn't seem to be working 🥲

![Screenshot_20230622-221208_Termius.png](https://github.com/Taxel/PlexTraktSync/assets/32086892/135fce15-3683-4af9-b066-63dcabd95d9d)

![Screenshot_20230622-221325_Termius.png](https://github.com/Taxel/PlexTraktSync/assets/32086892/1972c1c4-4ebf-4e9e-827c-565d1d667c79)

**Reply by glensc** [reply]
```yml
  - level: WARNING
    message: ":Friends-"
  - level: WARNING
    message: ":Brooklyn-"
  - level: WARNING
    message: ":The-Office-"
 ```

**Comment by glensc** [comment]
no response after last post. assume it got solved
